pkg/console/starter/starter.go

```diff
@@ -136,38 +128,34 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 
 	go consoleOperator.Run(ctx.StopCh)
 
-	// TODO: turn this back on!
-	// for now its just creating noise.... as we need to update library-go for it to work correctly
-	// our version of library-go has the old group
 	//clusterOperatorStatus := status.NewClusterOperatorStatusController(
-	//	controller.TargetNamespace,
-	//	controller.ResourceName,
-	//	// no idea why this is dynamic & not a strongly typed client.
-	//	dynamicClient,
-	//	&operatorStatusProvider{informers: consoleOperatorInformers},
+	//	api.ResourceName,
+	//	configClient.ConfigV1(),
+	//	operatorStatusProvider{informers: consoleOperatorInformers},
+	//	ctx.EventRecorder,
 	//)
-	//// TODO: will have a series of Run() funcs here
-	//go clusterOperatorStatus.Run(1, stopCh)
+
+	//go clusterOperatorStatus.Run(1, ctx.StopCh)
 
 	<-ctx.StopCh
 
 	return fmt.Errorf("stopped")
 }
 
-// I'd prefer this in a /console/status/ package, but other operators keep it here.
-type operatorStatusProvider struct {
-	informers externalversions.SharedInformerFactory
-}
-
-func (p *operatorStatusProvider) Informer() cache.SharedIndexInformer {
-	return p.informers.Console().V1alpha1().Consoles().Informer()
-}
-
-func (p *operatorStatusProvider) CurrentStatus() (operatorv1alpha1.OperatorStatus, error) {
-	instance, err := p.informers.Console().V1alpha1().Consoles().Lister().Consoles(api.TargetNamespace).Get(api.ResourceName)
-	if err != nil {
-		return operatorv1alpha1.OperatorStatus{}, err
-	}
-
-	return instance.Status.OperatorStatus, nil
-}
+//// I'd prefer this in a /console/status/ package, but other operators keep it here.
+//type operatorStatusProvider struct {
```

Comment:
It's unclear to me why you've commented the operator status out. Is this meant to be enabled in a follow on?

Reply:
Nope, turned it back on now, with the `.Console().V1().Consoles()...` instead of  `.Console().V1alpha1().Consoles()...`